### PR TITLE
PE-D Fixes

### LINKED ISSUE #158: [PE-D][Tester E] Using "Persons" instead of "Contacts"
Proposed Fix:

Listed all contacts

-------------
Labels: `type.FunctionalityBug` `severity.VeryLow`
original: yixiann/ped#2

### LINKED ISSUE #164: [PE-D][Tester B] Consistency in the verb form used in the description of the feature
`Sort contacts` can be changed to `sorting contacts` for consistency

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: PeiYee88/ped#3

### LINKED ISSUE #166: [PE-D][Tester E] Table of contents do not have consistent description of commands
viewing, editing, listing, adding 

v.s

sort 

Proposed fix:

Sorting all contacts

This is nearly a personal suggestion and not a bug

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: yixiann/ped#5

### LINKED ISSUE #167: [PE-D][Tester B] Consistency in the result display for list command
Perhaps can change `Listed all persons` to People instead for consistency?

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: PeiYee88/ped#4

### LINKED ISSUE #183: [PE-D][Tester E] Sorting by name may be incorrect
I feel A and AA should be together. Not sure why 'a' is between them.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: yixiann/ped#10

### LINKED ISSUE #186: [PE-D][Tester E] Unable to sort by phone.
Using 

"sort p/" does not yield any changes in the sorting
<i><video controls><source src="https://raw.githubusercontent.com/yixiann/ped/main/files/085b00f4-21b0-4040-ae19-b00895d45ebb.mov" type="video/mp4">Your browser does not support the video tag.</video><br>video:https://raw.githubusercontent.com/yixiann/ped/main/files/085b00f4-21b0-4040-ae19-b00895d45ebb.mov</i>

-------------
Labels: `type.FunctionalityBug` `severity.High`
original: yixiann/ped#11

### LINKED ISSUE #187: [PE-D][Tester C] Confusing index naming between people list and todo list
The user guide shows the use of the same name `INDEX` for indexes in people list and todo list. This is confusing for the readers. 

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: cadencjk/ped#4

### LINKED ISSUE #194: [PE-D][Tester B] Sorting command allows other inputs than the one specified in the User Guide
sort `p/` is the correct command. However, `sort p/1` and `p/amama` is accepted as well.

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: PeiYee88/ped#10

### LINKED ISSUE #197: [PE-D][Tester C] Application not accepting valid email
`thisIs..anEmail@gmail.com` throws an error, even though `.` is allowed in the local-part as described in the message.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: cadencjk/ped#8

### LINKED ISSUE #201: [PE-D][Tester C] Editing a todo tag clears existing tags
I would want to add a tag to todo, but editing the todo tag clears all existing tags. It would be troublesome to retype all existing tags all over again.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: cadencjk/ped#9

### PR DESCRIPTION
**Pull request by Eugene-Ong-W-X**
Clearing all 13 (including 2 duplicates) assigned bug reports / suggestions
- Excess issues (#191, #199, #212) will be linked in future PR

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -136,25 +138,38 @@ Example:
 
 ### Sorting contacts : `sort`
 
-Sort the list of contacts displayed by certain parameter(s).
+Sort the list of contacts displayed by 1 or more parameter(s) chosen by you.
```
Comment (ChongCheeKaiClarence) [inline docs/UserGuide.md]:
```suggestion
Sorts the list of contacts displayed by 1 or more parameter(s) chosen by you.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -136,25 +138,38 @@ Example:
...
+
+How *names (n/)*, *emails (e/)*, *addresses (a/ )* are sorted:
+* In alphabetical order. (e.g. `Al` comes before `Alfred` which comes before `Brad`)
+* Case differences are ignored. (e.g. `Al`, `al`, `AL`, and `aL` are identical when it comes to sorting)
```
Comment (ChongCheeKaiClarence) [inline docs/UserGuide.md]:
```suggestion
* Case insensitive. (e.g. `Al`, `al`, `AL`, and `aL` are identical when it comes to sorting)
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -136,25 +138,38 @@ Example:
...
+
+How *phone numbers (p/)* are sorted: 
+* In increasing numerical order. (e.g. `123` comes before `125` which comes before `1234`)
+
+How *tags (t/TAG)* are sorted:
+* Contacts with the *TAG* you specified will come before contacts without the *TAG*.
```
Comment (ChongCheeKaiClarence) [inline docs/UserGuide.md:151]: Just to clarify. Does this mean that those without the right `TAG` is in default sorting arrangement?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -309,10 +337,11 @@ Edits an existing todo in your SoConnect.
 
 Format: `todo edit INDEX [d/DESCRIPTION] [date/DATE] [pr/PRIORITY] [t/TAG]…​`
 
-* Edits the todo at the specified `INDEX`. The index refers to the index number shown in the displayed todo list. The index **must be a positive integer** 1, 2, 3, …​
+* Edits the todo at the specified `INDEX`.
 * At least one of the optional fields must be provided.
 * Parameters given will overwrite the existing values completely.
-  * For example, giving 1 tag in the edit command will erase all existing tags except the one given in the edit command.
+  * For example, giving 1 or more tag(s) in the edit command will erase all existing tags except the ones given in the edit command.
```
Comment (ChongCheeKaiClarence) [inline docs/UserGuide.md]:
```suggestion
  * For example, giving 1 or more tag(s) in the edit command will replace all existing tags with the ones given in the edit command.
```

**Review by ChongCheeKaiClarence — COMMENTED** [review]
Just some nits and comments. Otherwise, LGTM!

File: docs/UserGuide.md (hunk 3 above)
Comment (Eugene-Ong-W-X) [reply docs/UserGuide.md:151]: yea

**Pull request merged**